README.md

```diff
@@ -59,106 +21,25 @@ However none of these styles are enforced yet so go wild I guess.
 
 ```bash
 $ cp .env.j2 .env  # (first time only)
 ```
 
-The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
-
-You must also populate the MONGO_XXXXX variables with your Mongo account connection string, database and collection names.
-
-## Running the App
-
-Once the all dependencies have been installed, start the Flask app in development mode within the Poetry environment by running:
-
-```bash
-$ npm run dev
-```
-
-You should see output similar to the following:
+The `.env` file is used to set various environment variables.
 
-```bash
-$> dev
-$> concurrently "npm run css" "poetry run flask run"
-$[0]
-$[0] > css
-$[0] > npx tailwindcss -i ./todo_app/tailwind.css -o ./todo_app/static/css/index.css --watch
-$[0]
-$[1]  * Serving Flask app 'todo_app/app'
-$[1]  * Debug mode: on
-$[1] WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
-$[1]  * Running on http://127.0.0.1:5000
-$[1] Press CTRL+C to quit
-$[1]  * Restarting with stat
-$[0]
-$[0] Rebuilding...
-$[0]
-$[0] Done in 306ms.
-$[1]  * Debugger is active!
-$[1]  * Debugger PIN: 207-877-058
-$[1] 127.0.0.1 - - [05/Apr/2024 16:49:39] "GET / HTTP/1.1" 200 -
-$[1] 127.0.0.1 - - [05/Apr/2024 16:49:39] "GET /static/css/custom.css HTTP/1.1" 304 -
-$[1] 127.0.0.1 - - [05/Apr/2024 16:49:39] "GET /static/css/index.css HTTP/1.1" 200 -
-```
-
-Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
-
-Note: If you are getting "Insecure Transport Error" errors you can add: `OAUTHLIB_INSECURE_TRANSPORT=1` to your .env file for local dev.
-
-## Testing
-
-Unit tests & integration tests are implemented using [pytest](https://docs.pytest.org/). The full suite of tests can be run using:
-
-```bash
-$ poetry run pytest
-```
-
-or a specific test can be run using:
-
-```bash
-$ poetry run pytest path/to/test_file
-```
+You should populate all of the variables surrounded by handlebars e.g. `{{ example_variable }}` with the appropriate values, many of these can be found either in the GitHub project or the production app service on Azure.
 
 ## Docker containers
 
-It is also possible to run the app using Docker containers. There are two containers maintained in the Dockerfile, a production container and a development container. Both of which have corresponding services in the docker-compose.yml file making starting these containers easier for developers.
+We use docker containers for both development and testing of this app, this means you shouldn't need to install the relevant Node/Python packages but, while it wont be covered in this document, that should be possible to achieve.
```

Comment:
In practice this may make debugging Python code issues more challenging (as the VSCode launch configuration may no longer work with dependencies installed locally). The happy medium here is probably to setup a VSCode Dev container configuration (but that's beyond the scope of this exercise).